common/python/listener/base_jrpc_listener.py

```diff
@@ -163,26 +163,25 @@ def render_POST(self, request):
                 "request {0}: {1}".format(request.path, str(err)))
             return response
 
-# -----------------------------------------------------------------
-# -----------------------------------------------------------------
+    def start(self, host_name, port):
```

Comment:
I know this is existing code, but could you add a brief python """ docstring  """ comment?